lib/testing/gpuSimulateCudaUpchannelize.cpp

```diff
@@ -159,7 +159,7 @@ constexpr T sinc(const T x) {
 
 // array indexing
 
-constexpr int Eidx(int c, int d, int f, int p, int t) {
```

Comment:
It's weird that the function takes `c (specifying which component, `real` or `imag`, to return) and then ignores this argument. That argument probably shouldn't be there... But that's not for this PR. Maybe add a respective comment to the code instead?